drivers/mtd/mtd_nandecc.c

```diff
@@ -90,7 +88,7 @@ int nandecc_readpage(FAR struct nand_dev_s *nand, off_t block,
   unsigned int sparesize;
   int ret;
 
-  finfo("block=%d page=%d data=%p spare=%d\n",
+  finfo("block=%d page=%d data=%p spare=%p\n",
```

Comment:
Optional: we can change to use `PRIdOFF` for `block` instead of casting to `int`

Reply:
Done.